### PR TITLE
Add missing box shadow tokens

### PR DESCRIPTION
**Pull request by SaiSan-WMDE**
This makes available the component tokens needed to make error and warning states of inputs and dropdowns to display the correct focus.

Prep for issue: [T277033](https://phabricator.wikimedia.org/T277033)

Note: Not sure what those unrelated commits are doing there. Excuse my git.

### CODE REVIEW

File: vue-components/src/components/iconProps.ts [hunk 1]
```diff
@@ -13,6 +13,16 @@ export enum IconTypes {
 	SEARCH = 'search'
 }
 
-export const iconColors = [ 'base', 'warning', 'error', 'notice', 'success', 'inherit' ];
+export const iconColors = [
```
Comment (Ladsgroup) [inline vue-components/src/components/iconProps.ts:16]: This does look unrelated.

Comment (tzhelyazkova) [reply vue-components/src/components/iconProps.ts:16]: it's probably a VSCode linting extension being "smart"

File: vue-components/src/components/iconProps.ts [hunk 2]
```diff
@@ -13,6 +13,16 @@ export enum IconTypes {
...
+	'base',
+	'subtle',
+	'emphasized',
+	'inverted',
+	'warning',
+	'error',
+	'notice',
+	'success',
+	'inherit',
+];
```
Comment (tzhelyazkova) [inline vue-components/src/components/iconProps.ts:26]:
```suggestion
export const iconColors = [ 'base', 'warning', 'error', 'notice', 'success', 'inherit' ];
```

**Review by tzhelyazkova — APPROVED** [review]
I think this can be merged

File: vue-components/src/components/iconProps.ts (hunk 1 above)
Comment (tzhelyazkova) [reply vue-components/src/components/iconProps.ts:16]: Edit: I thought it's a VSCode extension being smart and fixing this but actually the line becomes too long hence lint fix.

Comment (SaiSan-WMDE) [reply vue-components/src/components/iconProps.ts:16]: I think this will be solved once the branch is updated. For some reason those changes to the icon component are being dragged into my PRs T-T
